.github/workflows/continuous_integration.yml

```diff
@@ -597,7 +598,7 @@ jobs:
     - name: Test opensim-core
       run: |
         cd $GITHUB_WORKSPACE/../build
-        ctest --parallel 4 --output-on-failure
+        ctest --parallel 4 --output-on-failure -E testMocoImplicit
```

Comment:
Rather than exclude the test manually here, you can add a flag to the Catch2 test definition to only exclude the test on Mac. [See CONTRIBUTING.md for details](https://github.com/opensim-org/opensim-core/blob/983f69d822649ab9312d2eb202c021dd0a12daf4/CONTRIBUTING.md?plain=1#L86).